celerybeatmongo/models.py

```diff
@@ -11,9 +11,9 @@
 
 
 def get_periodic_task_collection():
-    if hasattr(current_app.conf, "CELERY_MONGODB_SCHEDULER_COLLECTION") \
-            and current_app.conf.CELERY_MONGODB_SCHEDULER_COLLECTION:
-        return current_app.conf.CELERY_MONGODB_SCHEDULER_COLLECTION
+    if hasattr(current_app.conf, "mongodb_scheduler_collection") \
+            and current_app.conf.get("mongodb_scheduler_collection"):
+        return current_app.conf.get("mongodb_scheduler_collection")
```

Comment:
While this is the new style of naming, it would be preferable not to break users are using an older version of celery. I think it would be best to check both cases. It also looks like the method of getting data out has changed here, not sure if that's intentional or how that affects backwards compatibility.